tests/test_arguments.py

```diff
@@ -3,6 +3,11 @@
 import click
 from click._compat import PY2
 
+try:
+    import pathlib
+except ImportError:
+    pathlib = None
+
```

Comment:
Based on https://palletsproject.com/blog/ending-python2-support/ I suppose I should be removing the try/except?

Reply:
Don't worry about that until support is actually removed.